### PR TITLE
New Reader Detail: CSS fixes 

### PR DESCRIPTION
**Pull request by leandroalonso**
Review only after #14339 is merged.

This PR:

- Center images in small devices so they don't appear in the left if they're small
- Adds the color for the captions
- Adds a structure to refresh the CSS every two days if the user is online (won't affect offline)
- Only display images after they have been fully loaded, to simulate a native feel
- Do not display images broken, again, to have a native feel

To test

* Check this post, the image should be center-aligned: https://made4testing0318.blog/2020/04/17/another-image-block/
* In the same post, you can check the caption color
* Open a post full of images and check that they appear as a whole instead of slowly appearing
* Open a post with a broken image and check that the image doesn't appear, space it's there, but no broken images icons are displayed

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderCSS.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    private let now: Int = Int(Date().timeIntervalSince1970)
+
+    private let twoDays: Int = 60 * 60 * 24 * 2
+
+    /// Returns the Reader CSS appending a timestamp
+    /// We force it to update every 2 days
+    ///
+    var address: String {
+        guard let lastUpdated = store.object(forKey: "ReaderCSSLastUpdated") as? Int else {
```
[the remark this answers is not in the document]
Comment (leandroalonso) [reply WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderCSS.swift:16]: Done in de459f577106af7ab1b304a2d74f01fb59fbcf86

File: WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderCSS.swift [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            return url(appendingTimestamp: now)
+        }
+
+        // If the last time we fetched the CSS was 2 days ago and the user is online, refresh it
+        if now - lastUpdated >= twoDays && ReachabilityUtils.isInternetReachable() {
+            saveCurrentDate()
+            return url(appendingTimestamp: now)
+        } else {
+            return url(appendingTimestamp: lastUpdated)
+        }
```
[the remark this answers is not in the document]
Comment (leandroalonso) [reply WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderCSS.swift]: Done in 491f067aa89d27e0104466fab00fc8eb4b45c920

**Comment by leandroalonso** [comment]
@emilylaguna ready for another review!

> I was not able to get the images to load in as a whole, one of my tests the image didn't appear for many seconds after I opened the post: https://d.pr/v/Cohhzf

Regarding that, this is because the image is huge and it takes a lot of seconds to be processed in the server. That's why it takes longer.

> On the iPhone 5s running 11.4 there is a large amount of padding at the bottom of short posts, I don't see this present on 12.4 or 13.3: https://d.pr/v/ifH3KO

Fixed! And I also fixed a storyboard warning.

File: WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderCSS.swift [hunk 3]
```diff
@@ -7,32 +7,35 @@ struct ReaderCSS {
 
     private let now: Int = Int(Date().timeIntervalSince1970)
 
-    private let twoDays: Int = 60 * 60 * 24 * 2
+    private let expirationDays: Int = 5
+
+    private var expirationDaysInMs: Int {
```
Comment (emilylaguna) [inline WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderCSS.swift:12]: Is the result of this in milliseconds or seconds?

Comment (leandroalonso) [reply WordPress/Classes/ViewRelated/Reader/DetailWebView/ReaderCSS.swift]: Ops! Fixed in 8fabf2babb4f6266509708a6b962bbd82e97d6cc

**Pull request merged**